src/main/java/com/arondor/arender/alfresco/content/transformer/ContentTransformARenderRestClient.java

```diff
@@ -42,7 +42,7 @@ public float getWeatherPerformance()
         }
         catch (Exception e)
         {
-            LOGGER.info("Caught exception while trying to fetch weather score, is rendition server down?");
+            LOGGER.log(java.util.logging.Level.INFO, "Caught exception while trying to fetch weather score, is rendition server down?");
```

Comment:
Missing the exception in the method call.
```suggestion
            LOGGER.log(java.util.logging.Level.INFO, "Caught exception while trying to fetch weather score, is rendition server down?", e);
```